### PR TITLE
Adding keypair generation to cli.ts

### PR DESCRIPTION
**Pull request by ineiti**
Putting the keypair generation in typescript.

### CODE REVIEW

File: web/backend/src/cli.ts [hunk 1]
```diff
@@ -65,4 +66,15 @@ program
...
   });
 
+program
+  .command('keygen')
+  .description('Create a new keypair for the .env')
+  .action(() => {
+    const ed25519 = curve.newCurve('edwards25519');
+    const priv = ed25519.scalar().pick();
+    const pub = ed25519.point().mul(priv);
+    console.log(`PRIVATE_KEY=${priv}`);
```
Comment (emduc) [inline web/backend/src/cli.ts:76]: is it normal to simply print them instead of saving them where they are needed?

Comment (ineiti) [reply web/backend/src/cli.ts:76]: The goal here is to create the values that are needed for the `.env` file, so not to store them anywhere.

File: web/backend/readme.md [hunk 2]
```diff
@@ -10,18 +10,31 @@ Copy the database credentials from `config.env.template` to `/d-voting/bachend/s
...
+	"fmt"
+	"go.dedis.ch/kyber/v3/group/edwards25519"
+	"go.dedis.ch/kyber/v3/util/key"
+)
+
+func main() {
+	pair := key.NewKeyPair(&edwards25519.SuiteEd25519{})
+
+    fmt.Println("PUBLIC_KEY:", pubK)
 	fmt.Println("PRIVATE_KEY:", privK)
```
Comment (pierluca) [inline web/backend/readme.md:28]: tabs vs. spaces, I'd suggest making this uniform :)

File: web/backend/readme.md [hunk 3]
```diff
@@ -10,18 +10,31 @@ Copy the database credentials from `config.env.template` to `/d-voting/bachend/s
 
 ## Generate a keypair
 
-Here is a small piece of code to help generating the keys:
+Here is a small piece of code in go to generate the keypair for the backend:
```
Comment (pierluca) [inline web/backend/readme.md:13]:
```suggestion
Here is a small piece of code in Go to generate the keypair for the backend:
```

**Review by pierluca — COMMENTED** [review]
Minor remarks.

File: web/backend/src/cli.ts (hunk 1 above)
Comment (PascalinDe) [reply web/backend/src/cli.ts:76]: maybe add a line saying "please make sure to store this somewhere" as people might indeed expect the values to also be written to a file

Comment (ineiti) [reply web/backend/src/cli.ts:76]: Added an output, but now it won't be usable in scripts anymore...

**Review by emduc — APPROVED** [review]

**Review by pierluca — APPROVED** [review]

**Pull request merged**